projects/igniteui-angular/src/lib/radio/radio.component.ts

```diff
@@ -142,7 +142,7 @@ export class IgxRadioComponent implements ControlValueAccessor, EditorProvider {
      * @memberof IgxRadioComponent
      */
     @Input()
-    public value: any;
+    public value: string;
```

Comment:
Change the value type back to `any` because in strict template checks numeric fields will be rejected.